contracts/SpokePool.sol

```diff
@@ -651,7 +651,7 @@ abstract contract SpokePool is
     ) public payable override nonReentrant unpausedDeposits {
         // Check that deposit route is enabled for the input token. There are no checks required for the output token
         // which is pulled from the relayer at fill time and passed through this contract atomically to the recipient.
-        require(enabledDepositRoutes[inputToken][destinationChainId], "Disabled route");
+        if (!enabledDepositRoutes[inputToken][destinationChainId]) revert DisabledRoute();
```

Comment:
Is this done to emit the same revert reason from multiple places in the contract?

Reply:
reverts used to be cheaper on gas cost and bytecode but I'm not actually sure that's the case anymore. Regardless I do think its nice to re-use revert reasons and the name of the revert error can be descriptive and as long as you want while the require message is limited to 32 chars

Reply:
+1. I'm not sure how much an num costs bytecode-wise but a custom error is encoded like a function signature:

"Using a custom error instance will usually be much cheaper than a string description, because you can use the name of the error to describe it, which is encoded in only four bytes. A longer description can be supplied via NatSpec which does not incur any costs."

https://docs.soliditylang.org/en/latest/control-structures.html#revert